driver/config/ini_defines.h

```diff
@@ -45,9 +45,9 @@
 #endif
 
 #ifdef _win_
-#    define INI_TRACEFILE_DEFAULT "\\temp\\clickhouse-odbc.log"
+#    define INI_TRACEFILE_DEFAULT "\\temp\\clickhouse-odbc-driver.log"
```

Comment:
Not sure if it's a valid path for Windows. But not sure if  we can use env here, with smth like "%TEMP%\clickhouse-odbc.log"

Reply:
I believe, originally it was inspired by this:
https://github.com/lurcher/unixODBC/blob/4209bc229c64a0e00920dac635e666d2233c794f/include/sqlext.h#L194

Reply:
OK. We can check if it works. Someday :) Not important for now.